src/AutoScheduleUtils.h

```diff
@@ -47,8 +47,8 @@ int string_to_int(const std::string &s);
 /** Substitute every variable in an Expr or a Stmt with its estimate
  * if specified. */
 //@{
-Expr subsitute_var_estimates(Expr e);
-Stmt subsitute_var_estimates(Stmt s);
+Expr substitute_var_estimates(Expr e);
```

Comment:
I am in favor of fixing the spelling error in this name, but doing this sort of thing as a standalone PR in the future is probably better, to avoid confusion in reviewing.

Reply:
Sorry, was too lazy.